src/main/scala-2.11/org/apache/james/gatling/control/JamesWebAdministration.scala

```diff
@@ -0,0 +1,18 @@
+package org.apache.james.gatling.control
+
+import play.api.libs.ws.ning.NingWSClient
+
+object JamesWebAdministration {
+
+  val wsClient = NingWSClient()
+
+  val baseUrl: String = "http://172.17.0.4:8000"
```

Comment:
you should retrieve the target server through an env variable (and fallback to localhost)